src/services/user.ts

```diff
@@ -205,4 +229,36 @@ class UserParser {
 
         return notifications;
     }
+
+    public static parseMyReadLater(html: string): ReadLater[]{
+        const $ = cheerio.load(html);
+
+        const myReadLater: ReadLater[] = [];
+
+        $('div.fiction-list-item').each((i, el) => {
+            if($(el).hasClass('portlet')){ //skip if its ad
```

Comment:
see above

also maybe an utility function would be nice, i could see them changing this subtly and it would be easy to overlook changing it here as well